src/parser.js

```diff
@@ -734,8 +736,29 @@ export default class Parser {
         transition.id = methodValue.operationId;
       }
 
+      /* eslint-disable prefer-const */
+      let resourceParametersWithPath = [];
+      /* eslint-enable prefer-const */
+
+      _.forEach(resourceParameters, (resourceParameter, index) => {
```

Comment:
Any reason not to use a standard for of loop in ES6 instead of using lodash loop?